pkg/machine/e2e/init_windows_test.go

```diff
@@ -22,7 +26,23 @@ var _ = Describe("podman machine init - windows only", func() {
 		testDir, mb = setup()
 	})
 	AfterEach(func() {
-		teardown(originalHomeDir, testDir, mb)
+		teardownCb(originalHomeDir, testDir, mb, func() {
+			wslTask := func(command string, dist string) {
```

Comment:
This will need some if tests == wsl condition other it will run for hyperV as well.